test/api/v3/integration/tasks/challenges/POST-tasks_challenges_challengeId_tasks_id_score_direction.test.js

```diff
@@ -78,7 +78,7 @@ describe('POST /tasks/:id/score/:direction', () => {
       previousTaskHistory = task.history[0];
 
       expect(task.history).to.have.lengthOf(1);
-      expect(task.value).to.equal(1);
+      expect(task.value).to.equal(1.5);
```

Comment:
Let's save the original task value and just assert that the new task value is larger.